<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateEntInfoWidget</class>
 <widget class="QWidget" name="CreateEntInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>493</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>539</width>
    <height>493</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>539</width>
    <height>493</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelCompanyNotFound">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>110</y>
     <width>131</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>15</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;
color: #ff0000;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>未查询到该公司，请重新输入</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxArea">
   <property name="geometry">
    <rect>
     <x>424</x>
     <y>190</y>
     <width>98</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>98</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>98</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{border-radius:4px;color:rgba(51,51,51,1);border:1px solid rgb(222,222,222);font:14px 'Microsoft Yahei';padding-left:10px;}

QComboBox:focus{border:1px solid rgba(30,140,238,1);}

QComboBox::drop-down{subcontrol-origin:padding;subcontrol-position:top right;width:32px;border-radius:4px;}

QComboBox:down-arrow{image:url(:/ZcloudDesk/image/comboBoxDownArrow.png);}

QComboBox QAbstractItemView
{
	border-radius:0px;
	border: 1px solid rgb(161,161,161);
	outline: 0px;	
}
QComboBox QAbstractItemView::item 
{
	font:14px &quot;微软雅黑&quot;;
    height:32px;
}</string>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPhone">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>370</y>
     <width>368</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>368</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>368</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{border-radius:4px;color:rgba(51,51,51,1);border:1px solid rgb(222,222,222);font:14px 'Microsoft Yahei';padding-left:10px;}
QLineEdit:laceholder{color:rgba(51,51,51,1);}QLineEdit:focus{border:1px solid rgba(30,140,238,1);}
QLineEdit:disabled{background-color: rgb(247,248, 250);};</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>11</number>
   </property>
   <property name="placeholderText">
    <string>请输入负责人手机号</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPhoneError">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>410</y>
     <width>141</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>15</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;
color: #ff0000;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>请输入11位正确的手机号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>478</x>
     <y>15</y>
     <width>32</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>28</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>28</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image: url(:/EntCenterWidget/image/close.png);}
QPushButton:hover,pressed{border-image: url(:/EntCenterWidget/image/close_sel.png);};</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>25</y>
     <width>91</width>
     <height>14</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>56</width>
     <height>14</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>14</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>创建新企业</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>375</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>56</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>&lt;font color = red&gt;*&lt;/font&gt;&lt;font color =#333333&gt;负责人手机&lt;/font&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCity">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>190</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{border-radius:4px;color:rgba(51,51,51,1);border:1px solid rgb(222,222,222);font:14px 'Microsoft Yahei';padding-left:10px;}

QComboBox:focus{border:1px solid rgba(30,140,238,1);}

QComboBox::drop-down{subcontrol-origin:padding;subcontrol-position:top right;width:32px;border-radius:4px;}

QComboBox:down-arrow{image:url(:/ZcloudDesk/image/comboBoxDownArrow.png);}

QComboBox QAbstractItemView
{
	border-radius:0px;
	border: 1px solid rgb(161,161,161);
	outline: 0px;	
}
QComboBox QAbstractItemView::item 
{
	font:14px &quot;微软雅黑&quot;;
    height:32px;
}</string>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelLegalPeasonError">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>350</y>
     <width>131</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>15</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;
color: #ff0000;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>请输入2～15个任意字符</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxPro">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>190</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{border-radius:4px;color:rgba(51,51,51,1);border:1px solid rgb(222,222,222);font:14px 'Microsoft Yahei';padding-left:10px;}

QComboBox:focus{border:1px solid rgba(30,140,238,1);}

QComboBox::drop-down{subcontrol-origin:padding;subcontrol-position:top right;width:32px;border-radius:4px;}

QComboBox:down-arrow{image:url(:/ZcloudDesk/image/comboBoxDownArrow.png);}

QComboBox QAbstractItemView
{
	border-radius:0px;
	border: 1px solid rgb(161,161,161);
	outline: 0px;	
}
QComboBox QAbstractItemView::item 
{
	font:14px &quot;微软雅黑&quot;;
    height:32px;
}</string>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>315</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>56</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>&lt;font color = red&gt;*&lt;/font&gt;&lt;font color =#333333&gt;企业负责人&lt;/font&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCompanyName">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>70</y>
     <width>298</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>260</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>368</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{border-radius:4px;color:rgba(51,51,51,1);border:1px solid rgb(222,222,222);font:14px 'Microsoft Yahei';padding-left:10px;}
QLineEdit:laceholder{color:rgba(51,51,51,1);}QLineEdit:focus{border:1px solid rgba(30,140,238,1);}
QLineEdit:disabled{background-color: rgb(247,248, 250);};</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>请输入完整的公司名称后点击查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>75</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>56</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>&lt;font color = red&gt;*&lt;/font&gt;&lt;font color =#333333&gt;公司名称&lt;/font&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAddress">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>250</y>
     <width>368</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>368</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>368</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{border-radius:4px;color:rgba(51,51,51,1);border:1px solid rgb(222,222,222);font:14px 'Microsoft Yahei';padding-left:10px;}
QLineEdit:laceholder{color:rgba(51,51,51,1);}QLineEdit:focus{border:1px solid rgba(30,140,238,1);}
QLineEdit:disabled{background-color: rgb(247,248, 250);};</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>100</number>
   </property>
   <property name="placeholderText">
    <string>请输入街道、楼牌号等</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>140</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>56</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>&lt;font color = red&gt;*&lt;/font&gt;&lt;font color =#333333&gt;统一社会信用代码&lt;/font&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonOK">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>450</y>
     <width>72</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>72</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>72</width>
     <height>32</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(2, 164, 253, 1), stop:1 rgba(31, 139, 237, 1));border-radius:4px;font: 14px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}
			QPushButton:hover,pressed{ background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(40, 178, 253, 1),stop:1 rgba(64, 156, 240, 1));
		  border-radius:4px;font: 14px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}
QPushButton:disabled{ background-color:rgba(200,200,200,1);border-radius:4px;};</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLegalPeasonName">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>310</y>
     <width>368</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>368</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>368</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{border-radius:4px;color:rgba(51,51,51,1);border:1px solid rgb(222,222,222);font:14px 'Microsoft Yahei';padding-left:10px;}
QLineEdit:laceholder{color:rgba(51,51,51,1);}QLineEdit:focus{border:1px solid rgba(30,140,238,1);}
QLineEdit:disabled{background-color: rgb(247,248, 250);};</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
   <property name="placeholderText">
    <string>请输入企业负责人姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>255</y>
     <width>72</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>56</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>&lt;font color = red&gt;*&lt;/font&gt;&lt;font color =#333333&gt;详细地址&lt;/font&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSearch">
   <property name="geometry">
    <rect>
     <x>452</x>
     <y>70</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>72</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(2, 164, 253, 1), stop:1 rgba(31, 139, 237, 1));border-radius:4px;font: 14px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}
			QPushButton:hover,pressed{ background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(40, 178, 253, 1),stop:1 rgba(64, 156, 240, 1));
		  border-radius:4px;font: 14px &quot;微软雅黑&quot;;color:rgba(255, 255, 255, 1);}
QPushButton:disabled{ background-color:rgba(200,200,200,1);border-radius:4px;};</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSocietyCode">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>135</y>
     <width>368</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>368</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>368</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{border-radius:4px;color:rgba(51,51,51,1);border:1px solid rgb(222,222,222);font:14px 'Microsoft Yahei';padding-left:10px;}
QLineEdit:laceholder{color:rgba(51,51,51,1);}QLineEdit:focus{border:1px solid rgba(30,140,238,1);}
QLineEdit:disabled{background-color: rgb(247,248, 250);};</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelAddressError">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>290</y>
     <width>131</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>15</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;
color: #ff0000;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>请输入5～100个任意字符</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAreaError">
   <property name="geometry">
    <rect>
     <x>158</x>
     <y>230</y>
     <width>131</width>
     <height>15</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>15</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;
color: #ff0000;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>请选择完善的行政区域</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>195</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>56</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14px &quot;微软雅黑&quot;;
color:#333333;
border-bottom:0px;</string>
   </property>
   <property name="text">
    <string>&lt;font color = red&gt;*&lt;/font&gt;&lt;font color =#333333&gt;所在地区&lt;/font&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>450</y>
     <width>72</width>
     <height>32</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>72</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>72</width>
     <height>32</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:#FFFFFF;border-radius:4px;font:14px &quot;微软雅黑&quot;;color:#333333;border:1px solid #DEDEDE;}
QPushButton:hover,pressed{background-color:#F4F4F4;border-radius:4px;font:14px &quot;微软雅黑&quot;;color:#333333;}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
